src/main/views/viewManager.ts

```diff
@@ -80,25 +81,33 @@ export class ViewManager {
         server.tabs.forEach((tab) => this.loadView(srv, serverInfo, tab));
     }
 
-    loadView = (srv: MattermostServer, serverInfo: ServerInfo, tab: Tab, url?: string) => {
+    makeView = (srv: MattermostServer, serverInfo: ServerInfo, tab: Tab, url?: string): MattermostView => {
         const tabView = getServerView(srv, tab);
-        if (!tab.isOpen) {
-            this.closedViews.set(tabView.name, {srv, tab});
-            return;
-        }
         if (this.closedViews.has(tabView.name)) {
```

Comment:
Should this still be happening in `makeView` when it's a destructive action? Since you split up the old `loadView`, shouldn't this still happen in it or in `addView`? The lifecycle of a `MattermostView` is a bit confusing to me

Reply:
Agreed, both in the solution and in the confusion. I have elected to add this behaviour into `addView`, where it is most relevant.